compiler/luci/pass/src/QuantizationUtils.cpp

```diff
@@ -292,11 +292,9 @@ uint32_t cal_offset(loco::TensorShape &dimension, uint32_t *indices)
          indices[2] * dimension.dim(3).value() + indices[3];
 }
 
-uint32_t cal_offset_2d(loco::TensorShape &dimension, uint32_t *indices)
+uint32_t cal_offset_2d(loco::TensorShape &dimension, uint32_t indices[2])
```

Comment:
This is odd.
I don't like adding `2d` suffix and don't like adding `[2]` too.

Reply:
I understand that it's not easy to add it like that. 
I can use the original function`cal_offset`.

Reply:
And it would be better to split these two.
They don't have common context.